### PR TITLE
FEATURE: WEB-1991 Developer portal unified navigation for GD.UI

### PR DESCRIPTION
**Pull request by czechian**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/641/ : SUCCESS in 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/403/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/647/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/409/ : SUCCESS in 7s

File: website/pages/en/index.js [hunk 1]
```diff
@@ -73,9 +73,13 @@ class HomeSplash extends React.Component {
     return (
       <SplashContainer>
         <div className="left">
+          <img src="./img/homepage/icon-gooddata-ui.svg" />
           <ProjectTitle />
           <ProjectDescription />
-          <Button href={docUrl('about_gooddataui.html')} className="button-link">See documentation</Button>
+          <div className="buttonWrapper">
+            <Button href={docUrl('installation.html')} className="button button-dark">Get GoodData.ui</Button>
```
Comment (no23reason) [inline website/pages/en/index.js:80]: Should probably be `Get GoodData.UI` (uppercase UI).

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/648/ : SUCCESS in 1m 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/410/ : SUCCESS in 7s

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-gate-PR391

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-el7/12/ : SUCCESS in 1m 02s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/649/ : SUCCESS in 1m 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/85/ : FAILURE in 2m 34s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/178/ : SUCCESS in 18s

**Pull request merged**